pkg/services/ngalert/store/image_test.go

```diff
@@ -93,6 +94,64 @@ func TestIntegrationSaveAndGetImage(t *testing.T) {
 	}
 }
 
+type imagesByID []models.Image
+
+func (s imagesByID) Len() int           { return len(s) }
+func (s imagesByID) Less(i, j int) bool { return s[i].ID < s[j].ID }
+func (s imagesByID) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
+
+func TestIntegrationGetImages(t *testing.T) {
+	mockTimeNow()
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+	_, dbstore := tests.SetupTestEnv(t, baseIntervalSeconds)
+
+	// create an image foo.png
+	img1 := models.Image{Path: "foo.png"}
+	require.NoError(t, dbstore.SaveImage(ctx, &img1))
+
+	// GetImages should return the first image
+	imgs, err := dbstore.GetImages(ctx, []string{img1.Token})
+	require.NoError(t, err)
+	assert.Equal(t, []models.Image{img1}, imgs)
+
+	// create another image bar.png
+	img2 := models.Image{Path: "bar.png"}
+	require.NoError(t, dbstore.SaveImage(ctx, &img2))
+
+	// GetImages should return both images
+	imgs, err = dbstore.GetImages(ctx, []string{img1.Token, img2.Token})
+	require.NoError(t, err)
+	v := imagesByID(imgs)
+	sort.Sort(v)
+	assert.Equal(t, imagesByID{img1, img2}, v)
```

Comment:
You don't have to do the sorting yourself if you don't want to - does `assert.ElementsMatch` not work in this case?

Reply:
Thank you! This is what I was looking for but I couldn't find it!